platform/vpp/vppbld/Makefile

```diff
@@ -23,6 +23,7 @@ UID = $(shell id -u)
 GUID = $(shell id -g)
 
 VPP_REPO_DIR = repo
+VPP_CUSTOM_PLUGINS_DIR = ../plugins
```

Comment:
I think we should move plugins directory into vppbld, which is for building vpp. So everything for vpp build is under the same directory.

Reply:
Sounds good. I moved the plugins under vppbld in https://github.com/sonic-net/sonic-platform-vpp/pull/114/commits/935bb7992331e939ac92348a3ed225e1873aaa34.